website/docs/ai-ml/ray.md

```diff
@@ -6,317 +6,337 @@ sidebar_label: Ray on EKS
+We can then submit the job for PyTorch benchmark workload.
 
+```bash
+python job/pytorch_submit.py
 ```
 
-As shown above a model checkpoint is written to an S3 location so that it can be retrived for model serving. The location of the checkpoint is stored in the SSM Parameter Store.
+You can open http://localhost:8266 to monitor the progress of the pytorch benchmark.
 
-##### Ray Serve
+#### Teardown
 
-To create an inference endpoint which will be served using Ray Serve, create a Job to submit the Ray deployment `sources/serve_pytorch_huggingface_clothing.py`.
+:::caution
+To avoid unwanted charges to your AWS account, delete all the AWS resources created during this deployment.
+:::
 
+Destroy the Ray Clusters for pytorch followed by xgboost.
 
-```bash
-envsubst < sample-jobs/serve-pytorch-huggingface-clothing.yaml | kubectl create -f -
+From the pytorch directory.
 
+```bash
+terraform destroy -auto-approve
 ```
 
-##### Test Deployment
-
-A sample script `sources/pytorch_huggingface_clothing_client.py` is provided that uses the requests library to test the inference endpoint.
+From the xgboost directory.
 
 ```bash
-python sources/pytorch_huggingface_clothing_client.py
-
-Positive
+cd ../xgboost
+terraform destroy -auto-approve
 ```
 
-### Monitoring
+Delete the add-ons.
 
-This blueprint uses the `kube-prometheus-stack` to create a monitoring stack for getting visibility into your RayCluster.
-
-Open the Grafana dashboard using the url "https://ray-demo.bar.com/monitoring". The sample Ray dashboard can be accessed by browsing to the Ray grafana folder.
-
-![Ray Grafana Dashboard](img/ray-grafana.png)
+```bash
+cd ../../
+tf destroy -auto-approve -target=module.eks_blueprints_kubernetes_addons
+```
 
-## Cleanup
+Delete worker nodes, EKS cluster.
 
-To clean up your environment, destroy the Terraform modules in reverse order.
+```bash
+tf destroy -auto-approve -target=module.eks
+```
 
-Destroy the Kubernetes Add-ons, EKS cluster with Node groups and VPC
+Delete the VPC.
 
 ```bash
-terraform destroy -target="module.eks_blueprints_kubernetes_addons" -auto-approve
-terraform destroy -target="module.eks_blueprints" -auto-approve
-terraform destroy -target="module.vpc" -auto-approve
+tf destroy -auto-approve -target=module.vpc
 ```
 
-Finally, destroy any additional resources that are not in the above modules
+Delete everything else.
 
 ```bash
 terraform destroy -auto-approve
 ```
+
+### Future Work
+
+We want to enhance this blueprint to add functionality that makes this pattern more enterprise ready. To that end in the near future we will be adding enhancements around cost management using kubecost, better observability dashboards using OTEl, Amazon Managed Prometheus and Grafana, better security and data governance using OPA/Gatekeeper, IRSA, etc. Please let us know via [issues](https://github.com/awslabs/data-on-eks/issues) to see what you would like to see us build in this blueprint.
```

Comment:
Lets add this as a Note and move to the top of the doc

:::info

As part of our ongoing efforts to make this blueprint more enterprise-ready, we are actively working on adding several key functionalities. This includes cost management with Kubecost, advanced observability with OTEL, Amazon Managed Prometheus, and Grafana, as well as improved security and data governance using tools such as OPA/Gatekeeper and IRSA. If you have specific requirements or suggestions for this blueprint, please feel free to open an issue on our[ GitHub repository](https://github.com/awslabs/data-on-eks/issues).

:::